cmd/convox/deploy.go

```diff
@@ -51,9 +51,9 @@ func cmdDeploy(c *cli.Context) error {
 
 	switch a.Status {
 	case "creating":
-		return stdcli.Error(fmt.Errorf("app %s is still being created", app))
+		return stdcli.Error(fmt.Errorf("deployment blocked, %s is still being created, check `convox apps info`", app))
 	case "updating":
-		return stdcli.Error(fmt.Errorf("app %s is being updated", app))
+		return stdcli.Error(fmt.Errorf("deployment blocked, %s is already updating, check `convox apps info`", app))
```

Comment:
was gonna ask @nzoschke's thoughts on the same thing here, the important thing is pushing towards "check convox apps info"